app/src/main/kotlin/cz/covid19cz/erouska/receiver/ActionsReceiver.kt

```diff
@@ -0,0 +1,28 @@
+package cz.covid19cz.erouska.receiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import cz.covid19cz.erouska.service.CovidService
+import cz.covid19cz.erouska.utils.L
+
+class ActionsReceiver : BroadcastReceiver() {
+    companion object {
+        private const val PREFIX = "EROUSKA"
+        const val ACTION_PAUSE = "${PREFIX}_PAUSE"
+        const val ACTION_RESUME = "${PREFIX}_RESUME"
```

Comment:
The actions should be prefixed with the package name and in lower case. It's not super important, but good practice when you expose it to others.

Reply:
Is it? I didn't want to make it too long and hard to write. With the prefix, it seemed still unique enough to me :thinking: